app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,32 @@
+@import "foundation_and_overrides";
+@import "card.scss";
+@import url(https://fonts.googleapis.com/css?family=Quicksand|Raleway);
+
+body {
+  background-image: url("background-photo-stars.jpg");
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+}
+
+.top-bar { background-color: transparent; }
+.top-bar-section { background-color: transparent !important;  }
+.top-bar-section li { background-color: transparent !important;  }
+.top-bar-section li a { background-color: transparent !important;  }
```

Comment:
!important should not be used<br>Merge rule `.top-bar-section li a` with rule on line 14<br>Selector should have depth of applicability no greater than 2, but was 3